### ISSUE TITLE
Does `btfdump` support directly load btf from btf archives?

### ISSUE THREAD

**Issue by Officeyutong** [anakryiko/btfdump#6]
For example, directly load btf info from files like `/sys/kernel/btf/vmlinux`. Seems `btfdump` currently only supports read btf from ELF files.

**Comment by anakryiko** [comment]
Yep, it would be good to support that, but someone would need to actively work on this.

**Comment by anakryiko** [comment]
Implemented in https://github.com/anakryiko/btfdump/pull/16
